### ISSUE TITLE
Compilation fails

### ISSUE THREAD

**Issue by jinpark** [gretzky/react-native-android-wake-lock#1]
I tried running `./gradlew installDebug` and this module fails during compilation. Please let me know if you need more info.

```
> Task :react-native-android-wake-lock:compileDebugKotlin FAILED
.../node_modules/react-native-android-wake-lock/android/src/main/java/io/github/gretzky/wakelock/RNWakeLockPackage.kt: (22, 74): Expecting comma or ')'
.../node_modules/react-native-android-wake-lock/android/src/main/java/io/github/gretzky/wakelock/RNWakeLockPackage.kt: (22, 74): Expecting ')'
.../node_modules/react-native-android-wake-lock/android/src/main/java/io/github/gretzky/wakelock/RNWakeLockPackage.kt: (22, 75): Expecting member declaration
.../node_modules/react-native-android-wake-lock/android/src/main/java/io/github/gretzky/wakelock/RNWakeLockPackage.kt: (22, 76): Expecting member declaration
.../node_modules/react-native-android-wake-lock/android/src/main/java/io/github/gretzky/wakelock/RNWakeLockPackage.kt: (22, 77): Expecting member declaration
.../node_modules/react-native-android-wake-lock/android/src/main/java/io/github/gretzky/wakelock/RNWakeLockPackage.kt: (22, 79): Expecting member declaration
.../node_modules/react-native-android-wake-lock/android/src/main/java/io/github/gretzky/wakelock/RNWakeLockPackage.kt: (22, 83): Expecting member declaration
.../node_modules/react-native-android-wake-lock/android/src/main/java/io/github/gretzky/wakelock/RNWakeLockPackage.kt: (22, 84): Expecting member declaration
.../node_modules/react-native-android-wake-lock/android/src/main/java/io/github/gretzky/wakelock/RNWakeLockPackage.kt: (22, 95): Expecting member declaration
.../node_modules/react-native-android-wake-lock/android/src/main/java/io/github/gretzky/wakelock/RNWakeLockPackage.kt: (22, 96): Expecting member declaration
.../node_modules/react-native-android-wake-lock/android/src/main/java/io/github/gretzky/wakelock/RNWakeLockPackage.kt: (22, 97): Expecting member declaration
.../node_modules/react-native-android-wake-lock/android/src/main/java/io/github/gretzky/wakelock/RNWakeLockPackage.kt: (22, 99): Expecting member declaration
.../node_modules/react-native-android-wake-lock/android/src/main/java/io/github/gretzky/wakelock/RNWakeLockPackage.kt: (22, 100): Expecting member declaration
.../node_modules/react-native-android-wake-lock/android/src/main/java/io/github/gretzky/wakelock/RNWakeLockPackage.kt: (22, 101): Expecting member declaration
.../node_modules/react-native-android-wake-lock/android/src/main/java/io/github/gretzky/wakelock/RNWakeLockPackage.kt: (22, 103): Expecting member declaration
.../node_modules/react-native-android-wake-lock/android/src/main/java/io/github/gretzky/wakelock/RNWakeLockPackage.kt: (22, 103): Function declaration must have a name
.../node_modules/react-native-android-wake-lock/android/src/main/java/io/github/gretzky/wakelock/RNWakeLockPackage.kt: (22, 18): Return type of 'createViewManagers' is not a subtype of the return type of the overridden member '@NonNull public abstract fun createViewManagers(@NonNull p0: ReactApplicationContext): (Mutable)List<ViewManager<(raw) View!, (raw) ReactShadowNode<*>!>!> defined in com.facebook.react.ReactPackage'
.../node_modules/react-native-android-wake-lock/android/src/main/java/io/github/gretzky/wakelock/RNWakeLockPackage.kt: (23, 16): Type mismatch: inferred type is (Mutable)List<ViewManager<*, *>!>! but Unit was expected

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-android-wake-lock:compileDebugKotlin'.
> Compilation error. See log for more details

```

**Comment by lukefanning** [comment]
Yep unfortunately got the same error on RN 0.60.5.

**Comment by ravik2015** [comment]
Also getting the same issue for RN 0.62.0. Any idea what is missing here?

**Comment by redexp** [comment]
In file `android/src/main/java/io/github/gretzky/wakelock/RNWakeLockPackage.kt` should be `ReactApplicationContext` without brackets 
```
override fun createViewManagers(reactContext: ReactApplicationContext)
```

**Comment by DevQuayle** [comment]
> In file `android/src/main/java/io/github/gretzky/wakelock/RNWakeLockPackage.kt` should be `ReactApplicationContext` without brackets
> 
> ```
> override fun createViewManagers(reactContext: ReactApplicationContext)
> ```

You should have ```minSdkVersion = 23```. You can set this in ```<your-app-director>/android/build.gradle```

**Comment by ghost** [comment]
@gretzky Could you push a new release with your latest commit [Solving compilation error](https://github.com/gretzky/react-native-android-wake-lock/commit/89b8256a838c8dbab3687e6891baec4c6768a237) ?

**Comment by joshgachnang** [comment]
I published this as [react-native-wake-lock](https://www.npmjs.com/package/react-native-wake-lock) with the fix because I had a client who needed this package (thank you for the super useful package @gretzky!)

**Comment by vineyardbovines** [comment]
Sorry I missed this @jordan-tpz -- a new version has been released (v1.1.0)

@joshgachnang you can refer your client back to the original package if you'd like 😄 

My inbox was flooded with notifications so much so that these slip through the cracks, I've cleaned everything out now and will be able to see real notifications in my inbox going forward.
